src/app/_services/api/user-uqj.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable } from '@angular/core';
+import {environment} from '@environments/environment';
+import {HttpClient, HttpParams} from '@angular/common/http';
+import {Observable, of} from 'rxjs';
+import {catchError} from 'rxjs/operators';
+import {UQJ} from '@app/_models/uqj';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserUqjService {
```

Comment:
UQJ is the short form of UserQuestionJunction so the name of this service is UserUserQuestionJunction!
I would just rename it to UQJService